pkg/rotator/rotator.go

```diff
@@ -191,13 +202,15 @@ func addNamespacedCache(mgr manager.Manager, namespace string) (cache.Cache, err
 	if err != nil {
 		return nil, err
 	}
-	if err := mgr.Add(c); err != nil {
+	// Wrapping the cache to make sure it's also started when the manager
+	// hasn't been leader elected and CertRotator.RequireLeaderElection is false.
+	if err := mgr.Add(&cacheWrapper{Cache: c}); err != nil {
```

Comment:
how would someone set up this boolean?

Reply:
Ah good catch, I had local changes. Updated.